README.md

```diff
@@ -134,6 +134,43 @@ autocrlf = false
 ```
 Remove and clone again. This will prevent Windows git clients from automatically replacing Unix line endings LF with Windows line endings CRLF.
 
+#### Mac Troubleshooting
```

Comment:
I suspect this is not just a `Mac` issue. Maybe a more accurate title would be `Boot Troubleshooting`.